README.md

```diff
@@ -58,6 +64,124 @@ tests/                     - test cases package
 └── test_routes.py         - test suite for service routes
 ```
 
+## API Endpoints
+
+### Root Route (`/`)
+
+#### Description
+
+The root route provides metadata and basic information about the Promotion Service, such as its name, version, and the main endpoint. It serves as an entry point to verify that the API is accessible. This route can be used as a simple health check to ensure that the service is running properly.
+
+#### HTTP Method
+
+-   `GET`
+
+#### Example Request
+
+`GET /` 
+
+#### Response
+
+-   **Status Code**: `200 OK`
+-   **Content Type**: `application/json`
+-   **Response Body**:
+ 
+    `{
+      "service_name": "Promotion Service",
+      "version": "v1.0",
+      "endpoint": "/promotions"
+    }` 
+
+### Create a Promotion
+
+- **Method**: `POST`
+- **Description**: Creates a new promotion.
+-   **Response**: Returns the created promotion details with a unique promotion ID.
+
+### Read a Promotion
+
```

Comment:
Could you also add the URLs for the 'Create' and 'Read' operations? I would appreciate it!